dashboard/lib/queries/user/enabled_experiments.rb

```diff
@@ -0,0 +1,43 @@
+# Query object for retrieving enabled experiment names for a given user.
+#
+# Provides methods to get the current user's enabled experiments and
+# any experiments enabled for the user's associated teachers.
+#
+# Usage:
+#   query = Queries::User::EnabledExperiments.new(user)
+#   query.call            # => enabled experiment names for the user
+#   query.from_teachers   # => unique experiment names from user's teachers
+module Queries
+  module User
+    class EnabledExperiments < Queries::Base
+      def initialize(user)
+        @user = user
+      end
+
+      # Returns the enabled experiment names for the user.
+      #
+      # @return [Array<String>] list of experiment names
+      def call
+        experiment_names
+      end
```

Comment:
```suggestion
      def self.call(user)
        new(user).call
      end

      def initialize(user)
        @user = user
      end

      def call
        experiment_names
      end
```

This cleans up the invocation a bit by allowing you to just do: `Queries::User::EnabledExperiments.call(user)`

Reply:
Could we update the Base class to include the `self.call` class method so we only have to implement one `call` method in the new service classes?

Reply:
it already exists:
https://github.com/code-dot-org/code-dot-org/blob/eb1069d87ea9ad97273f02680938d638ddbaa1fd/dashboard/lib/queries/base.rb#L3-L5